app/clients/aws/client.py

```diff
@@ -6,6 +7,8 @@
 
 from app.model.aws_config import AWSConfig
 
+logger = logging.getLogger(__name__)
```

Comment:
Looks unused

Reply:
Good spot - trunk didn't catch it. Removed now.